### PR TITLE
Fix Tickets Emails Missing Footer

### PR DESCRIPTION
**Pull request by codingmusician**
### 🗒️ Description

This fixes the logic that was causing the footer not to appear in the email template.

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

### CODE REVIEW

File: src/views/v2/emails/template-parts/body/footer.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * @var string $header_text_color Header text color.
  */
 
-if ( empty( $footer_content ) || empty( $footer_credit ) ) {
+if ( empty( $footer_content ) && empty( $footer_credit ) ) {
```
Comment (juanfra) [inline src/views/v2/emails/template-parts/body/footer.php:23]: Even though, if any is empty, it should bail. Have you tested that's fixing the issue?

**Review by rafsuntaskin — APPROVED** [review]

File: src/views/v2/emails/template-parts/body/footer.php (hunk 1 above)
Comment (juanfra) [reply src/views/v2/emails/template-parts/body/footer.php:23]: I mean, if the content is empty OR the footer credit is set to OFF, it should bail.

Comment (codingmusician) [reply src/views/v2/emails/template-parts/body/footer.php:23]: And, yes. I tested this and it fixes the issue.

Comment (juanfra) [reply src/views/v2/emails/template-parts/body/footer.php:23]: Ahh, right! I thought the footer content was the content of the credit. The footer content is actually what's coming from the setting. Thanks for the clarification.

**Pull request merged**